### PR TITLE
feature: initialized cli tool with init and add_token features

### PR DESCRIPTION
**Pull request by TarunTomar122**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    destination_path_inference = os.path.join(root_dir, 'inference.py')
+    config_path_inference = os.path.join(root_dir, 'config.yaml')
+
+    # Copy the template file to the current directory
+    shutil.copyfile(inference_path, destination_path_inference)
+    shutil.copyfile(config_path, config_path_inference)
+
+    print("initialized neura-launch project")
+
+def add_token(args):
```
Comment (XanderWatson) [inline utils.py:23]: I feel it's better to add the token to the OS keyring than save it directly in a yaml file.

File: utils.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    token = args.token
```
Comment (XanderWatson) [inline utils.py:24]: Also, instead of taking the token through args, take it like a password, where it's not visible in the shell.

**Review by XanderWatson — CHANGES_REQUESTED** [review]
I've suggested a few changes. Additionally, according to the PEP8 style guide, your code should not exceed 80 characters in a single line. So kindly refactor it accordingly.

File: utils.py (hunk 2 above)
Comment (TarunTomar122) [reply utils.py:24]: I am wondering what others think of this @rohansingh9001 @kunxl-gg ... I believe this project-level token is not a very secret thing (or is it?) so I didn't care much about making it like a password.

Comment (rohansingh9001) [reply utils.py:24]: it is a secret. Any token should not be in code.

File: utils.py (hunk 1 above)
Comment (TarunTomar122) [reply utils.py:23]: totally can do that! this is also up to debate not sure what approach we want to go with... @rohansingh9001 shine ur light upon us

File: utils.py (hunk 2 above)
Comment (rohansingh9001) [reply utils.py:24]: @XanderWatson The modern line length limit is now 127. 72 was the old limit.

Comment (rohansingh9001) [reply utils.py:24]: Also yes, follow PEP8. The least one should do while making a python package.

To check for our rules use

`flake8 --max-line-length 127 .`

Comment (TarunTomar122) [reply utils.py:24]: yes, sory for not following PEP8 already! I'll fix that

Comment (TarunTomar122) [reply utils.py:24]: also about the project level token - I don't understand the issue with the current implementation. that looks very consistent to the existing patterns in other libraries...

**Review by XanderWatson — APPROVED** [review]
LGTM

**Pull request merged**